chapters/implementation/sections/game_objects_management/subsections/controller_classes.tex

```diff
@@ -1,5 +1,4 @@
-\section{Controllers}
-
+\subsection{\texttt{*Controller} classes}
```

Comment:
Why the *?

Reply:
To indicate that we have multiple controller classes. But maybe a better name for the section would be "IController interface", then we could write that these are the classes that implement it